README.md

```diff
@@ -39,7 +39,7 @@ Topics:
 * Create an enpoint called `/travel/mode` that returns the quickest method of travel given two locations, as well as the travel time. It should account for driving, walking, bicycling, and transit.
 
 ## Project Notes
-* You will need a separate key than what you've used for the Places API since it is actually a different API.
+* You will need a separate key than what you've used for the Places API since it is actually a different API. This can be found in the second link provided in *Resourses*.
```

Comment:
@EricHech ahem... _**"Resources"**_ 😉 

***

Also, if you want to make a link to that section in the readme, you can use "anchors" in markdown, like so:

> Anchors in Markdown
> To create an anchor to a heading in github flavored markdown. Add - characters between each word in the heading and wrap the value in parens (#some-markdown-heading) so your link should look like so:
> `[create an anchor](#anchors-in-markdown)`
> [create an anchor](#anchors-in-markdown)

...or you could use the URL:

https://github.com/LambdaSchool/Node-Express-Gmaps#resources-1

So, one way would be to modify line 42 like so:

```
You will need a separate key than what you've used for the Places API since it is actually a different API. This can be found in the second link provided in *[Resources](#resources-1)*.
```

NOTE: we use "resources-1" because it is the second instance of "Resources" as a header.

While we're here, it's never a bad time to consider grammar for readability. I might change the line like so:

```
You will need a separate key from the one(s) you've already used for the Places API since it is actually a different API. This can be found in the second link provided in *[Resources](#resources-1)*.
```

I dislike the use of "it" in the first sentence and "This" in the second. How would you word them to remove the ambiguity of those pronouns?